app/models/project_services/hipchat_service.rb

```diff
@@ -39,6 +39,8 @@ def fields
     [
       { type: 'text', name: 'token',     placeholder: 'Room token' },
       { type: 'text', name: 'room',      placeholder: 'Room name or ID' },
+      { type: 'checkbox', name: 'notify' },
+      { type: 'select', name: 'color', choices: ['yellow', 'red', 'green', 'purple', 'gray', 'random'] },
```

Comment:
Those are the colors that are suppored by [HipChat](https://www.hipchat.com/docs/apiv2/method/send_room_notification).